src/main/java/frc/robot/subsystems/shooterPivot/ShooterPivotIOSim.java

```diff
@@ -0,0 +1,49 @@
+package frc.robot.subsystems.shooterPivot;
+
+import edu.wpi.first.math.VecBuilder;
+import edu.wpi.first.math.system.plant.DCMotor;
+import edu.wpi.first.math.util.Units;
+import edu.wpi.first.wpilibj.simulation.SingleJointedArmSim;
+import frc.robot.Constants;
+
+public class ShooterPivotIOSim implements ShooterPivotIO {
+
+  private static final SingleJointedArmSim sim =
+      new SingleJointedArmSim(
+          DCMotor.getNEO(1),
+          Constants.ShooterPivotConstants.GEARING,
+          SingleJointedArmSim.estimateMOI(
+              Constants.ShooterPivotConstants.LENGTH_METERS,
+              Constants.ShooterPivotConstants.MASS_KG),
+          Constants.ShooterPivotConstants.LENGTH_METERS,
+          Units.degreesToRadians(Constants.ShooterPivotConstants.MAX_ANGLE_DEGREES - 10),
+          Units.degreesToRadians(Constants.ShooterPivotConstants.RETRACTED_ANGLE_DEGREES + 10),
+          Constants.ShooterPivotConstants.SIMULATE_GRAVITY,
+          Constants.ShooterPivotConstants.STARTING_ANGLE_RADS);
+
+  public ShooterPivotIOSim() {}
+
+  // TODO
+  @Override
+  public void setCurrentLimit(int currentLimit) {}
+
+  @Override
+  public void setPosition(double angleDeg) {
+    sim.setState(VecBuilder.fill(Units.degreesToRadians(angleDeg), 0.0));
+  }
+
+  @Override
+  public void setVoltage(double volts) {
+    sim.setInputVoltage(volts);
+  }
+
+  @Override
+  public void reseed(double absoluteEncoderVolts) {
+    sim.setState(
+        VecBuilder.fill(
+            Units.degreesToRadians(Constants.ShooterPivotConstants.RETRACTED_ANGLE_DEGREES), 0.0));
+  }
+
+  @Override
+  public void updateInputs(ShooterPivotInputs inputs) {}
```

Comment:
updateInputs() should be implemented (see [last year's code](https://github.com/FRC2713/Robot2023/blob/9c3afcb173141d6d85c83b49ddad71a4b11eded1/src/main/java/frc/robot/subsystems/fourBarIO/FourBarIOSim.java#L31-L59))